src/grdevicesxaml.cpp

```diff
@@ -20,10 +20,10 @@
 *
 * ***************************************************************************/
 
+#if 0
```

Comment:
This entire file needs to be reworked to compile with all the recent changes to R API handling. However, in light of https://github.com/Microsoft/RTVS/issues/3596, it doesn't make sense to do it now, since the fix to that bug will require more changes to the same things. Since we're not actually advertising this as a feature, disable it for now until https://github.com/Microsoft/RTVS/issues/3596 is done.